### ISSUE TITLE
Machine operators getting stuck on nonexistant recipes

### ISSUE THREAD

**Issue by Ciege666** [klikli-dev/occultism#688]
If a Machine Spirit places something in a crafting block, like water bucket in a furnace, it will insert the item and wait on it to complete the craft. Since its impossible to complete the craft it essentially freezes, removing the item(s) from the crafting block is the only way to reset the Machine spirit.

I was wondering it would be possible to have a timer on the crafting block?

After XX seconds the Machine Spirit checks to see if it has returned anything to the actuator/removed anything from the machine. If not, remove the items in question and return them to the storage actuator. I don't think it's a perfect solution but it would keep Machine Spirits running.

**Comment by klikli-dev** [comment]
That is an issue I've been aware of, but to be honest, I currently do not have the time and priority to implement a solution.
The proposed solution is solid, but needs proper implementation (e.g. configurable wait time or better, wait time that can be set per spirit, checks if we need edge case handling if the machine or storage actuator are no longer accessible, ..).

As the machine operators never were intended to be a full replacement for autocrafting I will set this as wontfix.
I might revisit it in the future because as I said, it is a solid solution, and if we do get a PR by someone annoyed enough to do it I will of course accept it :)
